component/attr_cache/attr_cache.go

```diff
@@ -802,7 +721,11 @@ func (ac *AttrCache) CopyFromFile(options internal.CopyFromFileOptions) error {
 		}
 		// TODO: Could we just update the size and mod time of the file here? Or can other attributes change here?
 		// TODO: we're RLocking the cache but we need to also lock this attr item because another thread could be reading this attr item
-		ac.invalidatePath(options.Name)
+
+		toBeUpdated, getErr := ac.cacheMap.get(options.Name) //empty for TestCopyFromFileDoesNotExist()
+		if getErr == nil {
+			ac.moveAttrCachedItem(toBeUpdated, options.Name, options.Name, time.Now())
+		}
```

Comment:
This doesn't seem right. If we're going to keep the cache entry, we need to know the new size.
If we can get that from options.File, that's great. But if we can't, we need to invalidate our cache entry.